### ISSUE TITLE
.me zone mistakenly tagged "withdrawn"

### ISSUE THREAD

**Issue by case** [zonedb/zonedb#899]
The nightly update job here -- https://github.com/zonedb/zonedb/commit/55ececbd2f86bd24fb9d3444ed1c5725ed560813 -- marked `.me` as tag `withdrawn` which is obviously not true.

We need to figure out why this happened.

**Comment by case** [comment]
A possibly-relevant log line from the nightly `Update` job.

```
2023-03-14T08:51:39.3045766Z [0m[0;33;49mRetired/withdrawn zone with active name servers: me
```

**Comment by case** [comment]
Also possibly relevant:

https://github.com/zonedb/zonedb/pull/881

**Comment by case** [comment]
Fixed in #900. Thanks @ydnar!
